src/modules/AuthenticationModule/components/Register/Register.tsx

```diff
@@ -1,10 +1,262 @@
+import { useContext, useState } from "react";
+import logo from "../../../../assets/PMS 3.png";
+import profileImgC from "../../../../assets/Group 48102075.png";
+
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { APIcontext } from "../../../../Context/baseUrl";
+import { ToastContext } from "../../../../Context/ToastContext";
 
 const Register = () => {
-  return (
-    <>
-      Register
-    </>
-  )
-}
-
-export default Register
+    const {baseUrl}=useContext(APIcontext)
+    const {getToastValue}=useContext(ToastContext)
+
+    const {
+        register,
+        handleSubmit,
+        watch,
+        formState: { errors },
+    } = useForm();
+    const [showPassword, setShowPassword] = useState(false);
```

Comment:
I don't like the name `showPassword` as it indicates to the reader that it is a function, not a state variable.
We change it to something like `[isPasswordShown,setIsPasswordShown]`. The new name encapsulates that it is a value and a boolean one.